templates/jlink/jlink-builder/jlink-builder-template.yaml

```diff
@@ -49,3 +58,31 @@ objects:
       to:
         kind: ImageStreamTag
         name: ubi9-openjdk-${JDK_VERSION}-jlink:latest
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
```

Comment:
I think we need to move to `Build` instead of `buildConfig`, yes (also in #471). I won't block merging this to `jlink-dev` for that reason, though, it goes on the TODO list.